app/views/merchant_items/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1><%= @merchant.name %>'s Items </h1>
-
+<h4><%= link_to "Create New Item", "/merchants/#{@merchant.id}/items/new" %></h4>
 <% @items.each do |item| %>
```

Comment:
you might want to wrap all of these items within a div so that your tests can look within this specific section, or look within a specific item.